sql_composer/pg/pg_translator.py

```
@@ -0,0 +1,591 @@
+            case PgDataTypes.UUID:
+                return f"'{self._escape_string(str(value))}'"
+            case _:
+                # Default case: treat as string
+                return f"'{self._escape_string(str(value))}'"
+
+    def where_to_sql(self, where: Where, column: Column) -> str:
+        # Convert all values to SQL expressions
+        values_as_pg_sql = [self.val_to_sql(column, value) for value in where.values]
+
+        if len(where.values) == 0:
+            raise ValueError(f"Operator {where.op} requires at least 1 value, got 0")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)